javascript/401-code-challenges/stack-and-queue/index.js

```diff
@@ -89,4 +89,32 @@ class Queue {
 }
 
 
-module.exports = { Stack, Queue };
+class PseudoQueue {
+  constructor() {
+    this.stack1 = new Stack();
+    this.stack2 = new Stack();
+  }
+
+  enqueue(value) {
+    while (this.stack1.length > 0) {
```

Comment:
your stack does not have a length property, so it is evaluating this as `undefined > 0` and skipping the code block altogether. Instead of adding a property, try making use of your `Stack.isEmpty()` method.